### PR TITLE
35944 - set entryPointUriPath and CustomApplicationId required

### PR DESCRIPTION
**Pull request by LiudmylaMasliuk**

### CODE REVIEW

File: paypal-commercetools-custom-application/src/constants.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 // Make sure to import the helper functions from the `ssr` entry point.
 import { entryPointUriPathToPermissionKeys } from '@commercetools-frontend/application-shell/ssr';
 
-export const entryPointUriPath = '${env:ENTRY_POINT_URI_PATH}';
```
Comment (realnaliboh) [inline paypal-commercetools-custom-application/src/constants.ts:4]: only out of sheer interest: why did you change this piece of code? Did you experience problems with permissions otherwise?

Comment (LiudmylaMasliuk) [reply paypal-commercetools-custom-application/src/constants.ts:4]: Mainly I have followed the guide https://docs.commercetools.com/custom-applications/deployment-examples/commercetools-connect but I think it is this way in the guides because of possible permissions issue.
